zmon_aws_agent/main.py

```diff
@@ -159,7 +168,8 @@ def main():
         elasticaches = aws.get_elasticache_nodes(region, infrastructure_account)
         dynamodbs = aws.get_dynamodb_tables(region, infrastructure_account)
         certificates = aws.get_certificates(region, infrastructure_account)
-        aws_limits = aws.get_limits(region, infrastructure_account, apps, elbs)
+        aws_limits = aws.get_limits(region, infrastructure_account, apps, elbs,
+                                    get_previous_limits(zmon_client, region, infrastructure_account))
```

Comment:
I don't think we need to make that call since we already have those entities in ``entities`` var.